ocdaction/ocdaction/templates/core/think.html

```diff
@@ -3,11 +3,116 @@
 {% load static %}
 
 {% block main %}
-    <section class="container">
-    	<div class="row">
-	    	<div class="col-xs-12">
-				<h1>Think</h1>
-				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse ac finibus ligula, at imperdiet nibh. Donec laoreet est vitae leo pulvinar, in porta risus viverra. Proin mollis diam sit amet lacus imperdiet ornare eu nec erat. Etiam semper lectus eget mi molestie hendrerit. Morbi vitae ornare odio. Phasellus vestibulum et ex bibendum commodo. Ut interdum egestas nisl at vestibulum. Suspendisse blandit lorem eros, feugiat consequat nibh vestibulum in. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis velit ante, pretium et arcu a, rutrum sodales sapien. Fusce vitae purus lacinia, laoreet enim at, consequat odio.</p>
+    <section>
+    	<div class="container">
+	    	<div class="row">
+		    	<div class="col-xs-12 col-md-8">
+					<h1 class="h1">Think</h1>
```

Comment:
Oh ok, I wasn't aware we use `.h1, .h2` classes to style headings.  Hmm I don't really see the reason why we do this?
It's a bit confusing, as on the Act page we also use `<h1>, <h2>` headings that are styled differently (since no class is used). If we'd like to have two different `<h2>` headings style, I'd rather use more semantic names so we avoid confusions why and where we use different styles (a good read about it: https://css-tricks.com/semantic-class-names/)

Since we'd rather want to use consistent style across the whole app let's to some small styles tidyup: could you please just remove heading classes and just style the html heading tags? It would require removing classes from all headings in`_typography.scss` partial and add `color: $grey;` to all of them (the colour is defined in `_colours.scss` partial).
It would look like this:

```
h1, h2, h3, h4, h5, h6 {
    color: $black-colour;
    font-family: $title-font-family;
    text-align: center;
}

h1 {
    color: $grey;
    font-size: 1.95em;
    font-weight: 400;
}
```

and the same for the rest of the headings (remove class and add color) 

That change will also be applied to 'Act' page so we will gain more consistency across the pages.
Let me know your thoughts or questions :)